src/library/objects/Quest.js

```diff
@@ -96,20 +96,28 @@ Quest Type:
 		}
 		return "";
 	};
-	
+
 	/* INCREMENT
 	Add one to tracking progress
 	@param {number} reqNum - index of counter type, mainly for Bw1. Default: 0
 	@param {number} amount - progress amount to be increased. Default: 1
 	@param isAdjustingCounter - if true, prevent recursively inc on specific quest itself
 	------------------------------------------*/
-	KC3Quest.prototype.increment = function(reqNum, amount, isAdjustingCounter){
+	KC3Quest.prototype.increment = function(reqNum=0, amount=1, isAdjustingCounter=false){
 		var self = this;
 		var isIncreased = false;
+		if (! ConfigManager.spCounterAdjust) {
+			if (this.tracking && this.isSelected()) {
+				if (this.tracking[reqNum][0] + amount <= this.tracking[reqNum][1]) {

```

Comment:
just wanna remind: this old logic is completely incorrect. eg: assume amount is 2, if tracking current value == max value - 1, current will never reach max value as `current + 2 > max`.
fortunately there is no any `increment` caller passing the `amount` argument before.
don't be angry over being a experimental animal, be pro lul